src/connectedk8s/azext_connectedk8s/_help.py

```diff
@@ -79,3 +79,31 @@
     - name: Show the details for a connected kubernetes cluster
       text: az connectedk8s show -g resourceGroupName -n connectedClusterName
 """
+
+helps['connectedk8s proxy'] = """
+  type: command
+  short-summary: Get access to a connected kubernetes cluster.
+  examples:
+  - name: Get access to a connected kubernetes cluster.
+    text: az connectedk8s proxy -n clusterName -g resourceGroupName
+"""
```

Comment:
There are more params here right?

Reply:
Thanks for pointing out. I have added examples for other params as well.